custom/icds_reports/reports/awc_reports.py

```diff
@@ -374,9 +375,13 @@ def get_data_for(date):
             age_filters,
             wfh_recorded_in_month_column(icds_feature_flag)
         )
+        agg_chm_config = copy.deepcopy(config)
+        if icds_feature_flag:
+            # Retrieving children of age 1-2 years
+            agg_chm_config['age_tranche'] = '24'
 
         queryset = AggChildHealthMonthly.objects.filter(
-            month=date, **config
+            month=date, **agg_chm_config
```

Comment:
such how it will be applied to other indicators as well apart from immunization which we don't want

Reply:
this method is only being used here